express-post-json/server.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+
+const app = express();
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = Object.values(grades);
+  res.json(gradesArray);
+});
+
+app.use(express.json());
```

Comment:
Although this middleware is only used by the routes below this, it can safely be used on the `get` route. This middleware will do nothing if the body is not JSON or if there is no body. So it's common to just put this at the top, right after creating the express app.

We will be using several more middlewares, so having them all in one place at the top is convenient (when they can be at the top).